Orange/widgets/unsupervised/owmds.py

```diff
@@ -200,6 +200,7 @@ def __init__(self):
 
         self.embedding = None  # type: Optional[np.ndarray]
         self.effective_matrix = None  # type: Optional[DistMatrix]
+        self.stress = None
```

Comment:
As shown by failing tests, this conflicts with the static method with the same name.
This results in a crash when setting the size of points to "Stress".

Reply:
It's not that I don't warn students about this exact mistake.